cl/opinion_page/templates/court.html

```diff
@@ -3,100 +3,119 @@
 {% load static %}
 
 {% block title %}
-  Decisions for {{ court }} – CourtListener.com
+	Decisions for {{ court }} – CourtListener.com
 {% endblock %}
 {% block og_title %}
-  Decisions for {{ court }} – CourtListener.com
+	Decisions for {{ court }} – CourtListener.com
 {% endblock %}
 {% block description %}
-  Search and read decisions from the {{ court }}
+	Search and read decisions from the {{ court }}
 {% endblock %}
 {% block og_description %}
-  Search and read decisions from the {{ court }}
+	Search and read decisions from the {{ court }}
 {% endblock %}
 
 {% block navbar-o %}active{% endblock %}
 {% block sidebar %}{% endblock %}
 
 
 {% block footer-scripts %}
-  <script src="{% static "js/jquery.bootstrap-growl.js" %}"></script>
+	<script src="{% static "js/jquery.bootstrap-growl.js" %}"></script>
 
 {% endblock %}
 
 {% block content %}
-  <div id="court_homepage" class="col-xs-12">
-    <div class="row">
-      <div class="col-xs-4">
-        <img src="{% static 'img/' %}{{ pk }}-court.jpg" class="img-responsive" width="460" height="206" >
-      </div>
-      <div class="col-xs-8">
-        <h1>Decisions of the {{ court }}
-        </h1>
-        <div class="lead gray">
-          <p>The Supreme Judicial Court of Maine has partnered with the non-profit organization, Free Law Project, to host its court decisions on CourtListener.com. Free Law Project is proud to support this partnership and makes the decisions available online as they are released by the court.</p>
-        </div>
-      </div>
-    </div>
+	<div id="court_homepage" class="col-xs-12">
+		<div class="row">
+			<div class="col-xs-4">
+				<img src="{% static 'img/' %}{{ pk }}-court.jpg" class="img-responsive"
+					 width="460" height="206">
+			</div>
+			<div class="col-xs-8">
+				<h1>Decisions of the {{ court }}
+				</h1>
+				<div class="lead gray">
+					<p>The Supreme Judicial Court of Maine has partnered with the
+						non-profit organization, Free Law Project, to host its court
+						decisions on CourtListener.com. Free Law Project is proud to
+						support this partnership and makes the decisions available
+						online as they are released by the court.</p>
```

Comment:
this also just looks like white space changes -can we please not add whitespace where its not needed